src/service.js

```diff
@@ -348,6 +348,28 @@ export function getParentWindowFrameElement(node, topWin) {
 
 
 /**
+ * Gets the resource ID of the amp-ad element containing the passed node.
+ * If there is no containing amp-ad tag, then null will be returned.
+ * TODO(jonkeller): Investigate whether non-A4A use case is needed. Issue 11436
+ * @return {?string}
+ */
+export function getAmpAdResourceId(node, topWin) {
```

Comment:
nit: this helper method is too specific for `service.js`. If you look at other methods in this file, they are all much general. `ad-helper.js` may be a better place for this function. But I think what we have before is good enough.  WDYT?

Reply:
It's needed by amp-ad-exit.js and amp-analytics.js, so I figured I shouldn't repeat it.
What is your preference - both of these locations, or ad-helper.js?

Reply:
: ) I am fine with both.

Reply:
Ok, will move to ad-helper. Thanks!

Reply:
Done.